photon-core/src/main/java/org/photonvision/vision/processes/VisionModuleChangeSubscriber.java

```diff
@@ -218,8 +241,9 @@ public void onDataChangeEvent(DataChangeEvent<?> event) {
                     logger.error("Unknown exception when setting PSC prop!", e);
                 }
 
-                parentModule.saveAndBroadcastSelective(wsEvent.originContext, propName, newPropValue);
+                parentModule.saveAndBroadcastSelective(originContext, propName, newPropValue);
```

Comment:
I know save is buffered/cheap (we just request a save, we don't trigger one), but does it make "sense" to ask for a save every single time? It seems like this should work fine but idk how ""right"" it is

Reply:
I'm not so sure about this either. I know it is working atm though.

Reply:
eh send it